tools/bionano/.shed.yml

```diff
@@ -0,0 +1,9 @@
+categories:
+- Assembly
```

Comment:
Cris can you make a suite out of this please, in case we need to add more tools.

Reply:
Sure!